source/providers.rst

```diff
@@ -80,15 +83,18 @@ Packages for Ubuntu 16.04 and CentOS 7 are also available.
 Requirements for production
 ```````````````````````````
 
-* Powerful-enough Oneprovider
+* Powerful enough Oneprovider
```

Comment:
```suggestion
* Oneprovider
```

Reply:
It goes without saying that the Oneprovider resources need to be "powerful enough".

Reply:
The fact is that it needs quite uncommon specifications (lots of CPU, RAM and even an SSD ideally), but I agree that's not very clear, will try to improve this.

Reply:
Implemented, you were right.